internal/kuberesource/resourcegen/main.go

```diff
@@ -42,18 +49,17 @@ func main() {
 		var err error
 		switch set {
 		case "coordinator":
-			subResources = kuberesource.CoordinatorBundle()
+			subResources = kuberesource.PatchRuntimeHandlers(kuberesource.CoordinatorBundle(), runtimeHandler)
 		case "runtime":
 			subResources, err = kuberesource.Runtime(platform)
-			if err != nil {
-				log.Fatalf("Error generating runtime: %v", err)
-			}
 		case "openssl":
-			subResources = kuberesource.OpenSSL()
+			subResources = kuberesource.PatchRuntimeHandlers(kuberesource.OpenSSL(), runtimeHandler)
 		case "emojivoto":
-			subResources = kuberesource.Emojivoto(kuberesource.ServiceMeshDisabled)
+			subResources, err = kuberesource.Emojivoto(kuberesource.ServiceMeshDisabled)
+			subResources = kuberesource.PatchRuntimeHandlers(subResources, runtimeHandler)
```

Comment:
What if the first call fails and subResources is nil/empty?

Reply:
`range` over `nil` / empty should do nothing. So `subResources` will just stay `nil` / empty.